python_modules/libraries/dagster-aws/dagster_aws/ecs/launcher.py

```diff
@@ -1,173 +0,0 @@
-from dagster import Bool, Field, Noneable, StringSource, check
-from dagster.cli.api import ExecuteRunArgs
-from dagster.core.launcher import RunLauncher
-from dagster.core.storage.pipeline_run import PipelineRun
-from dagster.serdes import ConfigurableClass, ConfigurableClassData, serialize_dagster_namedtuple
-
-from .client import ECSClient
-
-
-class ECSRunLauncher(RunLauncher, ConfigurableClass):
```

Comment:
I agree that most of this can be removed. Would it be possible to keep a skeleton of the ECSRunLauncher that simply instantiates the new ECSClient with a `launch_run` method that calls `run_task`?

Reply:
I could, but I don't think it would represent be a working `RunLauncher` 😞.

I have [another branch in the works](https://github.com/jmsanders/dagster/pull/new/jordan-ecs-set-task) that adds additional methods to the the `ECSClient` that I think _should_ be sufficient to flesh out a simple `ECSRunLauncher`. I plan to have a PR up for that branch Monday. Mind if I wait until then (or perhaps even one more branch after that) to add the `ECSRunLauncher` back in?

Reply:
Once this PR is merged, I'll switch the comparison on https://github.com/jmsanders/dagster/pull/1 to the main dagster repo. These changes should be (mostly?) sufficient to get a basic `ECSRunLauncher` implemented.

Reply:
Got it, your plan sounds good to me!